dashboard/config/database.yml

```diff
@@ -51,6 +51,7 @@ mysql_defaults: &mysql_defaults
     username: <%= writer.user || 'root' %>
     password: <%= writer.password || '' %>
     host: <%= writer.host || 'localhost' %>
+    port: <%= writer.port || 3306 %>
```

Comment:
Since the proxysql service listens to a non-standard port (to coexist properly with local mysql itself), we now need to explicitly add the port (with fallback to the default 3306) in our database configurations.